packages/design-system/src/globals.css

```diff
@@ -16,7 +16,9 @@
     -ms-overflow-style: none; /* IE and Edge */
     scrollbar-width: none; /* Firefox */
   }
-
+  body {
+		@apply dark:bg-black  min-h-screen relative;
+	}
```

Comment:
we shouldn't apply this styles to the library, if we do this wherever the library is used it is going to change the page background. I recommend to do this in the storybook related config files instead